components/brave_rewards/resources/request_ads_enabled/lib/client.ts

```diff
@@ -38,6 +38,9 @@ export function createClient (): Client {
 
     enableAds () {
       port.postMessage(true)
+      window.setTimeout(() => {
```

Comment:
Nit: it's typical to access `setTimeout` directly (rather than via a property access on `window`).